easygcm/src/main/java/eu/inloop/easygcm/GcmHelper.java

```diff
@@ -185,30 +186,15 @@ void registerInBackground(final Context context, final RegistrationListener regi
         final AsyncTask<Void, Void, Void> registrationTask = new AsyncTask<Void, Void, Void>() {
```

Comment:
Some simple synchronization is in place, there is a mRegistrationRunning atomicboolean, and next registrations will be skipped if there is already an registration running. Synchronization is not needed maybe - we should just make sure that we don't run more at once.